github/admin_orgs.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2013 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import "context"
+
+// createOrgRequest is a subset of Organization and is used internally
+// by Create to pass only the known fields for the endpoint
+//
+type createOrgRequest struct {
+	Login *string `json:"login"`
+	Admin *string `json:"admin"`
+}
+
+// CreateOrg creates a new organization in Github Enterprise.
+//
+// Note that only a subset of the org fields are used and org must
+// not be nil
+//
+// Github Enterprise API docs: https://developer.github.com/enterprise/2.16/v3/enterprise-admin/orgs/#create-an-organization
```

Comment:
Can you please use the version-agnostic URL for GitHub Enterprise links?  See other files in repo for examples.